src/boxFunctions/boxEnclosingPoints.ts

```diff
@@ -0,0 +1,33 @@
+import { IVec, boxReset } from "..";
+
+/**
+ * Computes the smallest bounding box that contains all of the provided points.
+ *
+ * If the provided array is empty, this method returns a box with all values set
+ * to `NaN`.
+ *
+ * @param points the points to contain
+ */
+export function boxEnclosingPoints(...points: IVec[]) {
```

Comment:
Thoughts on making this an interleaved list of coordinates? I.e. `...coordinates: number[]` as in `[x0, y0, x1, y1, x2, y2, ...]`. It's _slightly_ more consistent with other examples in the library, i.e. for polygons/polylines, but may be more awkward to consume.

Reply:
Could you make this method take an `out: IBox = boxAlloc()` param and writing to that value instead of returning a new box always?